public/components/application-ui-dividers/3.html

```diff
@@ -0,0 +1,4 @@
+<span class="flex items-center">
+    <span class="px-4">Align Left</span>
```

Comment:
Should use `pr` instead of `px` and let's use `pr-6` so it is consistent spacing with the first one.